storybook/pages/AmountValidatorPage.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Validators 0.1
+
+Item {
+    id: root
+
+    ColumnLayout {
+        anchors.centerIn: parent
+
+        TextField {
+            id: textField
+
+            Layout.alignment: Qt.AlignHCenter
+
+            validator: AmountValidator {
+                decimalPoint: buttonGroup.checkedButton.text
+
+                maxIntegralDigits: maxIntegralDigitsSpinBox.value
+                maxDecimalDigits: maxDecimalDigitsSpinBox.value
+            }
+        }
+
+        Label {
+            Layout.alignment: Qt.AlignHCenter
+
+            text: `acceptableInput: ${textField.acceptableInput}`
+        }
+
+        ButtonGroup {
+            id: buttonGroup
+
+            buttons: radioButtonsRow.children
+        }
+
+        RowLayout {
+            id: radioButtonsRow
+
+            Layout.alignment: Qt.AlignHCenter
+
+            RadioButton {
```

Comment:
Maybe adding some labels for these radio buttons? Didn't see the text without reading the code and I've thought the buttons are some leftovers:D